packages/fxa-content-server/app/scripts/views/mixins/sync-choice-mixin.js

```diff
@@ -7,22 +7,40 @@
  *
  * @mixin DoNotSync mixin
```

Comment:
I think `sync-optional-mixin` might be a little clearer since we typically refer to this feature as that.